src/applications/appeals/10182/pages/evidenceIntro.js

```diff
@@ -22,6 +22,11 @@ const contactInfo = {
         },
         enableAnalytics: true,
         classNames: 'vads-u-margin-top--0',
+        widgetProps: {
+          // ID from evidenceUploadIntroDescription
+          Y: { 'aria-describedby': 'additional-evidence-description' },
```

Comment:
Is it going to read Yes for `Y` and No for `N`?

Reply:
The `widgetProps` targets the radio based on the `value` which is either `Y` or `N`.